src/strings/en-us.json

```
@@ -131,7 +131,7 @@
     "ButtonUninstall": "Uninstall",
     "ButtonUseQuickConnect": "Use Quick Connect",
     "ButtonWebsite": "Website",
-    "Bwdif": "BWDIF",
+    "Bwdif": "BWDIF - Bob Weaver DeInterlacing Filter",
```

The hardware acceleration options are formatted differently, better to keep it consistent.

```suggestion
    "Bwdif": "Bob Weaver DeInterlacing Filter (BWDIF)",
```

---

OK. Understood. Done.